OliveVSIX/NugetPacker/NugetPackerLogic.cs

```diff
@@ -95,24 +95,13 @@ static void GenerateNugetFromVSProject(string projectAddress)
 
             if (TryPackDotnet(projectAddress, out string packingMessageDotnet))
             {
-                if (!TryPushDotnet(packageFilename, out string pushingMessage))
```

Comment:
There was no need to separate pushing NuGet package by `dotnet` or `nuget.exe`. when the NuGet package is generated there is no difference between pushing by `dotnet` or `nuget.exe`. _this change was related to this pull request: [149](https://github.com/Geeksltd/Olive/pull/149)_